BookingsApi/BookingsApi.DAL/Commands/RemoveHearingCommand.cs

```diff
@@ -35,7 +35,7 @@ public async Task Handle(RemoveHearingCommand command)
                 .Include(x => x.HearingCases).ThenInclude(x => x.Case)
                 .Include(x => x.Participants).ThenInclude(x => x.Person).ThenInclude(x => x.Organisation)
                 .Include(x => x.Participants).ThenInclude(x => x.Questionnaire).ThenInclude(x => x.SuitabilityAnswers)
-                .Include(x => x.Participants).ThenInclude(x => x.LinkedParticipants)
+                .Include(x => x.Participants).ThenInclude(x => x.LinkedParticipants).ThenInclude(x => x.Participant)
                 .Include(x => x.Endpoints).ThenInclude(x => x.DefenceAdvocate)
                 .Include(x => x.Participants).ThenInclude(x => x.LinkedParticipants)
```

Comment:
Do we need LinkedParticipants twice? Lines 38 and 40

Reply:
and line 68, can you change it to IsNullOrEmpty?

Reply:
replaced with a select many

Reply:
cheers